docs/content/webcomponents.js

```diff
@@ -51,21 +53,25 @@ function Navigation({next, previous}) {
         </div>`;
 }
 
-function FantomasSetting({name, green, orange, red, gr}) {
+function FantomasSetting({name, green, orange, red, gr, deprecated}) {
```

Comment:
Oh whoops, I thought I removed this. Since we weren't giving them their own sections I nixed this whole concept, I just forgot to remove some stuff from the web components 🤦🏻‍♂️